pkg/api/handlers/wellknown/handler.go

```diff
@@ -18,6 +18,7 @@ func SetupHandlers(baseURL string, config services.OAuthAuthorizationServerConfi
 
 	mux.HandleFunc("GET /.well-known/oauth-protected-resource/mcp-connect/{mcp_id}", h.oauthProtectedResource)
 	mux.HandleFunc("GET /.well-known/oauth-authorization-server/{mcp_id}", h.oauthAuthorization)
+	mux.HandleFunc("GET /.well-known/oauth-authorization-server/mcp-connect/{mcp_id}", h.oauthAuthorization)
```

Comment:
imo a comment should be left here explaining why we did this

Reply:
Done.